nsdb-core/src/main/scala/io/radicalbit/nsdb/actors/ShardReaderActor.scala

```diff
@@ -342,6 +370,19 @@ class ShardReaderActor(val basePath: String, val db: String, val namespace: Stri
 
   override def receive: Receive = readOps
 
+  private def retrieveField(values: Seq[Bit],
```

Comment:
The two methods below are quite cryptic. It got a while to figure out what's their purpose. Maybe It would be better to add comments, but it's not mandatory for this PR.